examples/language/gpt/test_ci.sh

```diff
@@ -0,0 +1,3 @@
+if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
```

Comment:
There is no need to check for the existence of `requirements.txt` since it is inside the GPT folder  for sure?

Reply:
oh..for other folder such as images/vit, does not exist...?
![image](https://user-images.githubusercontent.com/99854690/210469212-0fc5dbda-83c5-49b0-8d2b-367696c67e93.png)

Reply:
but this `test_ci.sh` is inside the `gpt` folder?

Reply:
Well, I see. Every folder should have this. Without this is can be seen as a bug..

Reply:
Not really? Some examples can have `requirements.txt` for pip, some examples can have `environement.yaml` for conda and some examples can have either of them. It should be up to the author of the example to decide how they want to install their dependencies.